proto/teleport/lib/teleterm/v1/access_request.proto

```diff
@@ -14,11 +14,11 @@
 
 syntax = "proto3";
 
-package teleport.terminal.v1;
+package teleport.lib.teleterm.v1;
 
 import "google/protobuf/timestamp.proto";
 
-option go_package = "github.com/gravitational/teleport/lib/teleterm/v1";
+option go_package = "github.com/gravitational/teleport/gen/proto/go/teleport/lib/teleterm/v1";
```

Comment:
I thought I'd be able to completely get rid of go_package and let the files be saved to `gen/proto/go`, like we do with JS files in `buf-js.gen.yaml`.

However, after getting rid of it, protoc-go-grpc was complaining saying that I need to either add `go_package` to protos or add the `M` flag to the command, pointing me here https://developers.google.com/protocol-buffers/docs/reference/go-generated#package.

Reply:
`go_package` should be present and explicit, this is alright (desired, even).